src/app_service/service/saved_address/service.nim

```diff
@@ -1,23 +1,27 @@
 import chronicles, sequtils, json
-
+import eventemitter
 import ./service_interface, ./dto
-
 import status/statusgo_backend_new/saved_addresses as backend
 
 export service_interface
 
 logScope:
   topics = "saved-address-service"
 
+const SIGNAL_SAVED_ADDRESS_CHANGED = "savedAddress/changed"
+export SIGNAL_SAVED_ADDRESS_CHANGED
```

Comment:
FYI, you can just put a `*` next to the variable name and it is considered exported.

Reply:
TIL: I didn t know it worked on constants. Updated